### PR TITLE
[DOCS] Amend the Install and Prerequisites sections

### PR DESCRIPTION
**Pull request by ologvinova**
The commit also fixes the noticed typos and grammar.

### CODE REVIEW

File: docs/install.md [hunk 1]
```diff
@@ -1,34 +1,59 @@
...
+This section describes how to install k0smotron on top of an existing k0s
+Kubernetes cluster that allows for creation and management of the k0s
+control plane.
 
-```bash
-kubectl apply -f https://docs.k0smotron.io/{{{ extra.k0smotron_version }}}/install.yaml
-```
+# Hardware requirements
+
+k0smotron does not require any special hardware aside from the one required for
```
Comment (makhov) [inline docs/install.md:9]: Could we emphasize here, that k0s requirements here is for workload clusters, not for k0smotron itself?

**Review by makhov — APPROVED** [review]

File: docs/install.md [hunk 2]
```diff
@@ -1,34 +1,62 @@
...
-    url: "https://github.com/k0sproject/k0smotron/releases/latest/infrastructure-components.yaml"
-    type: "InfrastructureProvider"
+kubectl apply -f https://docs.k0smotron.io/{{{ extra.k0smotron_version }}}/install.yaml
 ```
 
-Once you have the configuration file in place, you can use clusterctl to create a cluster:
+Before creating your first control planes, select the required use case described in
+[k0smotron usage](usage-overview.md).
+
+# Per-module installation for Cluster API
```
Comment (juanluisvaladas) [inline docs/install.md:50]: I think this bit here would make more sense in the prerequisites document

**Comment by ologvinova** [comment]
Abandoning this PR for the sake of https://github.com/k0sproject/k0smotron/pull/498

**Comment by ologvinova** [comment]
Abandoning this PR for the sake of https://github.com/k0sproject/k0smotron/pull/498

**Pull request closed without merging**